tests/rptest/services/utils.py

```diff
@@ -93,6 +101,12 @@ def _get_hostname(self, host) -> str:
         """
         return ""
 
+    @staticmethod
+    def _get_badlines_exception(bad_loglines):
+        """Method to raise correct Exception class.
+        """
+        return BadLogLines(bad_loglines)
+
```

Comment:
it looks like this can be dropped, right? i don't see any overloads.

Reply:
Yes, right now it makes no sense anymore.